src/shims/intrinsics.rs

```diff
@@ -427,6 +427,16 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             "atomic_max_rel" => this.atomic_min_max(args, dest, false, AtomicRwOp::Release)?,
             "atomic_max_acqrel" => this.atomic_min_max(args, dest, false, AtomicRwOp::AcqRel)?,
             "atomic_max_relaxed" => this.atomic_min_max(args, dest, false, AtomicRwOp::Relaxed)?,
+            "atomic_umin" => this.atomic_min_max(args, dest, true, AtomicRwOp::SeqCst)?,
```

Comment:
What are these about? If "u" is for "unsigned", then the code needs to be different... `<` behaves differently in the same bit patterns depending on whether the values are signed or unsigned.

Reply:
;) I was just getting to that, you are fast on the review

Reply:
:D

Please make sure to add a test that does the max of `-1` and `1` (signed) and the min of `1` and `usize::MAX` (unsigned), to make sure the code uses the signs correctly.